ast/src/parsed/mod.rs

```diff
@@ -320,7 +320,7 @@ impl<R> Children<Expression<R>> for EnumVariant<Expression<R>> {
 #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Serialize, Deserialize, JsonSchema)]
 pub struct TraitDeclaration<E = u64> {
     pub name: String,
-    pub type_vars: TypeBounds,
+    pub type_vars: Option<Vec<String>>,
```

Comment:
I'm thinking, does it make sense to have no type variables? Then we can have exactly one implementation of the trait.

So it sounds like it's better to have them non-optional and non-empty (already ensured by the parser).